src/_layouts/documentation.erb

```diff
@@ -6,10 +6,19 @@ layout: application
 <%= render Shared::Sidebar.new(data: @entries, current: resource) do |sidebar| %>
   <% if resource.data.sidebar_title.present? %>
     <% sidebar.slot :before do %>
-      <h1>
+      <h2>
         <%= resource.data.sidebar_title %>
-        <span class="version"><%= resource.data.sidebar_version %></span>
-      </h1>
+        <%# TODO: move in dedicated component %>
```

Comment:
This can be done in dedicated commit, and then fix the missing slot in this PR only